src/sign-in-winuiapp/README.md

```diff
@@ -0,0 +1,97 @@
+---
+-->
+<!-- SAMPLE ID: DOCS-CODE-027-->
+# .NET (C#) | WinUI 3 Packaged Desktop App | user sign-in, protected web API access (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This .NET WinUI 3 Packaged Desktop App authenticates a user and then makes a request to the Graph API as the authenticated user. The response to the request is presented to the user.
+
+![A screenshot of a WinUI 3 Packaged Desktop App displaying a response from Microsoft Graph.](./app.png)
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Visual Studio 2022, [configured with WinUI 3 workload and components](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=vs-2022-17-1-a%2Cvs-2022-17-1-b#required-workloads-and-components)
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the application.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting  | Value for this sample app                                           | Notes                                                                           |
+|--------------------------------:|:--------------------------------------------------------------------|:--------------------------------------------------------------------------------|
+| **Name**                        | `active-directory-dotnet-winui3`                                    | Suggested value for this sample. <br/> You can change the app name at any time. |
+| **Supported account types**     | **Accounts in this organizational directory only (Single tenant)**  | Suggested value for this sample.                                                |
+| **Platform type**               | **Mobile and desktop applications**                                 | Required value for this sample                                                  |
+| **Redirect URIs**               | `https://login.microsoftonline.com/common/oauth2/nativeclient`      | Required value for this sample                                                  |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Open the project in Visual Studio
+
+Next, open the _MsalExamples.csproj_ project in Visual Studio.
+
+### 3. Update code sample in _MainWindow.cs_ with app registration values
+
+Finally, set the following values in _MainWindow.xaml.cs_.

```

Comment:
Should these both be _MainWindow.cs_ or is it intentional for them to be different?